datacollection/openapi/v2/schema/schema-uam-adsb.yaml

```diff
@@ -1,366 +0,0 @@
-info:
-    version: 'v1.0'
-    title: ADS-B Out
-    description: >-
-      OpenAPI 2.0 compliant definitions for ADS-B Out data.
-swagger: "2.0"
-paths:
-  '/no-paths':
-    put:
-      responses:
-        '410':
-          description: >-
-            Gone. Not implemented. Will not be implemented. Path included for
-            proper use by automated tooling.
-definitions:
-    ADSBObject:
-      type: object
-      required:
-        - aircraft
-      properties:
-        aircraft:
-          type: array
-          items:
-            $ref:
-              '#/definitions/ADSBModel'
-
-    ADSBModel:
-      type: object
-      required:
-        - icaoAddress
-      properties:
-        icaoAddress:
-          type: string
-          pattern: '^[0-9a-fA-F]{6}$'
-          description: ICAO of the aircraft
-        trafficSource:
-          type: integer
-          enum: [0,1]
-          description: 0 = 1090, 1 = UAT
-        latDD:
-          type: number
-          description: Latitude expressed as decimal degrees
-        lonDD:
-          type: number
-          description: Longitude expressed as decimal degrees
-        altitudeMM:
-          type: number
-          description: Geometric altitude or barometric pressure altitudein millimeters
-        headingDE2:
-          type: number
-          description: Course over ground in centi-degrees
-        horVelocityCMS:
-          type: integer
-          description: Horizontal velocity in centimeters/sec
-        verVelocityCMS:
-          type: integer
-          description: Vertical velocity in centimeters/sec with positive being up
-        squawk:
-          type: number
-          pattern: '^[0-7]{4}$'
-          description: Squawk code
-        altitudeType:
-          type: integer
-          enum: [0,1]
-          description: Altitude Source, 0 is pressure, 1 is geometric
-        callsign:
-          type: string
-          pattern: '^[A-Z0-9]+ *$'
-        emitterType:
-          type: number
```

Comment:
I found a error in codegen where the enum was type=number but needed to be integer.    In any case we can be consistent by changing all enums to integer